server/kepler_model_trainer.py

```diff
@@ -231,7 +231,7 @@ def return_model_weights(model_type):
             #Categorical Label and Weights Dict
             categorical_weights_dict = create_categorical_vocab_weights_relation(categorical_labels, list_of_all_categorical_labels_vocab, list_of_all_categorical_labels_weights)
             # Combine all features and weights into a single dictionary
-            final_dict = {"All_Weights": {"Numerical_Variables": numerical_weights_dict, "Categorical_Variables": categorical_weights_dict, "Bias_Weight": bias} }
+            final_dict = {"All_Weights_{}".format(model_type): {"Numerical_Variables": numerical_weights_dict, "Categorical_Variables": categorical_weights_dict, "Bias_Weight": bias} }
```

Comment:
could you keep the same format as before? so that we do not have to repeatedly declare very similar struct in kepler side. adding "core" or "dram" suffix limits extensibility and reusability